### PR TITLE
Adjusting css styles

### PR DESCRIPTION
**Pull request by Daphne210**
Screenshot (84)

### CODE REVIEW

File: src/components/workspace/chatbot/chatbot-button.component.tsx [hunk 1]
```diff
@@ -49,7 +49,7 @@ const ChatbotButton = () => {
       {showMessage && (
         <div className={styles.messageContainer}>
           <span className={styles.chatbotText}>
-            Hello {provider}, Welcome to UgandaEMR Plus Chatbot
+            Hello {provider}, Welcome to iFHIS Chatbot
```
Comment (jabahum) [inline src/components/workspace/chatbot/chatbot-button.component.tsx]: can you make this configurable so that we dont have to hard code the system name

**Review by jabahum — COMMENTED** [review]
Thanks

File: src/components/workspace/chatbot/chatbot-button.component.tsx (hunk 1 above)
Comment (Daphne210) [reply src/components/workspace/chatbot/chatbot-button.component.tsx]: This has been resolved @jabahum

**Review by jabahum — APPROVED** [review]
Thanks @Daphne210

**Pull request merged**